UICatalog/Scenarios/HexEditor.cs

```diff
@@ -16,21 +16,30 @@ public class HexEditor : Scenario
     private HexView _hexView;
     private MenuItem _miAllowEdits;
     private bool _saved = true;
-    private StatusItem _siPositionChanged;
+    private Shortcut _siPositionChanged;
     private StatusBar _statusBar;
 
-    public override void Setup ()
+    public override void Main ()
     {
-        Win.Title = GetName () + "-" + _fileName ?? "Untitled";
+        Toplevel app = new Toplevel ()
```

Comment:
Please add `Application.Init ();` into the scenario to avoid exception been throwing.